odoo/models.py

```diff
@@ -5152,7 +5118,7 @@ def browse(self, ids=None):
             res.partner(7, 18, 12)
 
         :param ids: id(s)
-        :type ids: int or list(int) or None
+        :type ids: int or iretable(int) or None
```

Comment:
```suggestion
        :type ids: int or iterable(int) or None
```

Reply:
Change it in (the right PR for the clean): https://github.com/odoo/odoo/pull/79563